build.gradle

```diff
@@ -59,7 +59,8 @@ dependencies {
             'org.springframework.boot:spring-boot-starter-data-jpa',
             'org.springframework.boot:spring-boot-starter-security',
             'io.jsonwebtoken:jjwt-impl:0.11.2',
-            'org.codehaus.groovy:groovy-all:3.0.5'
+            'org.codehaus.groovy:groovy-all:3.0.5',
+            'com.fasterxml.jackson.core:jackson-databind:2.12.5'
     )
```

Comment:
Do we really need additional libraries, if the 'port problem' was resolved?

Reply:
The problem was resolved and didn't depend on this library